README.md

```diff
@@ -158,9 +158,12 @@ DESCRIPTION:
   specified. If not specified, a new wallet address will be generated.
 
 OPTIONS:
-   --channel value  the 8-byte compact channel ID of the channel to loop out (default: 0)
-   --addr value     the optional address that the looped out funds should be sent to, if left blank the funds will go to lnd's wallet
-   --amt value      the amount in satoshis to loop out (default: 0)
+   --channel value               the 8-byte compact channel ID of the channel to loop out (default: 0)
+   --addr value                  the optional address that the looped out funds should be sent to, if left blank the funds will go to lnd's wallet
+   --amt value                   the amount in satoshis to loop out (default: 0)
+   --conf_target value           the number of blocks from the swap initiation height that the on-chain HTLC should be swept within (default: 6)
+   --max_swap_routing_fee value  the max off-chain swap routing fee in satoshis, if let blank a default max fee will be used (default: 0)
+   --fast                        Indicate you want to swap immediately, paying potentially a higher fee. If not set the swap server might choose to wait up to 30 minutes before publishing the swap HTLC on-chain, to save on chain fees. Not setting this flag might result in a lower swap fee.
```

Comment:
The 30 min may be dynamic, I might not mention it in a static comment

Reply:
This is already hard-coded in the description of the flag. This just represents the current output of the `loop out --help` command. I think we should change this text once the command itself doesn't have a static value anymore.